.github/workflows/ci-run-test.yml

```diff
@@ -140,6 +140,10 @@ jobs:
           } >> "$GITHUB_ENV"
           # For debugging:
           ps aux
+      - name: Install tsc on windows and macos
+        if: ${{ runner.os == 'windows' || runner.os == 'macOS' }}
+        run: |
+          npm install typescript -g
```

Comment:
Does this ship on linux?

Reply:
It seems like :thinking: I mostly went of off test failures, which don't seem to happen on Linux but installing this is needed for both windows and macos.